visualization/README.md

```diff
@@ -7,27 +7,44 @@
 ## Jump to Section
 
 -   [Installation](#installation)
--   [Tasks](#tasks)
+-   [Development Tasks](#tasks)
+-   [Testing](#testing)
 -   [JSON structure](#json-structure)
 -   [License](LICENSE.md)
 
-## Installation
+# Installation
 
-Change working directory `cd <projectpath>/codecharta/visualization/`
+You can start with **Codecharta Visualization** on multiple ways:
 
-#### Install application for desktop usage with npm
+-   Installation as a [npm package](#npm-package) (Recommended)
+-   Run an operating system specific [standalone](#run-a-standalone)
+-   Build it yourself from the github [repository](#build-it-yourself)
+-   Try out the [online version](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json&file=codecharta_analysis.cc.json)
 
--   install with `npm install codecharta-visualization -g`
--   run with `codecharta-visualization`
+## NPM Package
 
-#### Install application for desktop usage
+> Make sure you have [Node.js](https://nodejs.org/en/download) installed <br>
```

Comment:
I would add the min version we need to install.

Reply:
Thanks will do that 👍